source_files/coal/c_compile.cc

```diff
@@ -50,18 +50,18 @@ static constexpr const char *punctuation[] =
      "^",  "(",  ")",  "-",  "+",  "=",  "[",  "]",  "{",   "}",  ".", "<", ">", "#", "&", "|", nullptr};
 
 // simple types.  function types are dynamically Allocated
-static Type type_void     = {ev_void};
-static Type type_string   = {ev_string};
-static Type type_float    = {ev_float};
-static Type type_vector   = {ev_vector};
-static Type type_function = {ev_function, &type_void};
-static Type type_module   = {ev_module};
-static Type type_null     = {ev_null};
+static Type type_void     = {ev_void, nullptr, 0, {nullptr}};
```

Comment:
It is too bad C++ doesn't seem to like {0};  maybe something to look into one more time before landing?

Reply:
Looks like -Wextra is the only thing marking these, maybe -Wall on its own suffices?

Reply:
Will address when the new PR comes up that uses Wextra + suppression of the field initializer stuff in these cases